tf_encrypted/keras/layers/convolutional.py

```diff
@@ -101,29 +103,39 @@ def __init__(self,
     self.strides = strides
     self.padding = padding.upper()
     self.data_format = data_format
+    if activation is not None:
+      logging.info("Performing an activation before a pooling layer can result "
```

Comment:
we should find a better way, but currently you should create a logger object and pass it the right name (see eg config.py)